packages/stylelint-plugin/src/rules/no-prefix-literal.ts

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright 2021 Palantir Technologies, Inc. All rights reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Root, Result } from "postcss";
+import parser from "postcss-selector-parser";
+import stylelint from "stylelint";
+import type { Plugin, RuleTesterContext } from "stylelint";
+
+import { checkImportExists } from "../utils/checkImportExists";
+import { insertImport } from "../utils/insertImport";
+
+const ruleName = "@blueprintjs/no-prefix-literal";
+
+const messages = stylelint.utils.ruleMessages(ruleName, {
+    expected: (unfixed: string, fixed: string) => `Use the \`${fixed}\` variable instead of the \`${unfixed}\` literal`,
+});
+
+const bannedPrefixes = ["bp1", "bp2", "bp3", "bp4"];
```

Comment:
i would just ban bp, bp3, and bp4. the first 2 don't exist afaik

Reply:
👍